test/data.go

```diff
@@ -85,9 +90,12 @@ func (a AppUserSlice) Less(x, y int) bool {
 func (a AppUserSlice) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
 
 type App struct {
-	Id       string `gorm:"primaryKey;column:application_id"`
-	Name     string
-	TenantId string `gorm:"primaryKey;column:org_id"`
+	Id        string `gorm:"primaryKey;column:application_id"`
+	Name      string
+	TenantId  string `gorm:"primaryKey;column:org_id"`
+	CreatedAt time.Time
```

Comment:
use gorm.Model instead similar to protostoreMsg

Reply:
gorm.Model itself has "Id" as primary key, which has conflict with parent struct's Id field.

Reply:
aah ok